trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -45,13 +44,11 @@ const getCaseDeal = async (req, res) => {
 
   const { submissionType } = deal.dealSnapshot;
 
-  const dealCancellationIsEnabled = isDealCancellationEnabled(submissionType, user);
-  let hasDraftCancellation = false;
+  const dealCancellation = await api.getDealCancellation(dealId, userToken);
 
-  if (dealCancellationIsEnabled) {
-    const cancellation = await api.getDealCancellation(dealId, userToken);
-    hasDraftCancellation = !isEmpty(cancellation);
-  }
```

Comment:
Checking the status rather than using `isEmpty` is soooo much nicer - maybe too much for this PR but there's a few other places I added this check that could now look for draft (and that would prevent the pages being accessed once the cancellations submitted)

Reply:
Yes I noticed that! Let's do that as a separate PR